examples/with-jest/.babelrc

```diff
@@ -1,15 +1,5 @@
 {
-  "env": {
-    "development": {
-      "presets": ["next/babel"]
-    },
-    "production": {
-      "presets": ["next/babel"]
-    },
-    "test": {
-      // next/babel does not transpile import/export syntax.
-      // So, using es2015 in the beginning will fix that.
-      "presets": ["es2015", "next/babel"]
-    }
-  }
+  "presets": [
```

Comment:
If we don't get this, we've to use a crazy `.babelrc` like this to support Jest.